### DISCUSSION TITLE
Impossible to sign in to GitHub using a Mobile Browser and the GitHub Android app

### DISCUSSION THREAD

**Discussion by chimbori** [community/community#141226] [Mobile]
When attempting to login to a 3P website using GitHub login, GitHub on Mobile Web fires off a two factor request to launch the GitHub Mobile app, but there's no code displayed in the Mobile Web browser to enter into the GitHub Mobile app.

Here’s a video illustrating the issue.

https://github.com/user-attachments/assets/6e8a7b22-c6e3-4601-adf8-1102914432fd

**Comment by alcere** [comment]
Hi @chimbori. We are sorry you're having trouble signing in. Can you tell us which website you're trying to access? Or does this happen when the website asks you to verify your identity using GitHub 2FA? Thank you!

**Reply by chimbori** [reply]
I added a video to explain, and I updated the title to reflect that it isn’t related to 3P OAuth; I can’t even sign in to GitHub itself using the browser.

**Reply by alcere** [reply]
I see, we are very sorry for this inconvenience. Could you please share the Android version of your device with us?

**Reply by chimbori** [reply]
Android 14 Build AP2A.240905.003

**Reply by alcere** [reply]
Thanks @chimbori. I've created an internal issue, and we will investigate ways to get this fixed. We are sorry for the troubles.

**Reply by chimbori** [reply]
Hello, any updates you can share?

**Reply by skyzea1** [reply]
@alcere  I have same problem, any update on this?

**Reply by alcere** [reply]
We apologize for the delay in identifying the cause of the error. Our team is actively trying to find a solution, and we truly appreciate your patience as we resolve the issue.

**Reply by mibressler** [reply]
same here

**Reply by chidimo** [reply]
Same problem for over a week now. Can we just get the option to use an authentication code at least, whe this issue is being figured out?

There are PRs I need to look at on my phone from time to time.

**Comment by Himani0002** [comment]
I am a full stack development student and BCA.

Tell me about any internship or anything you might know.

please

**Comment by iskae** [comment]
Hey folks 👋 

We released an update to Beta last week, which will be available in Production tomorrow. Can you please test the latest version and let us know if it resolved login issues for you? 

Cheers!

**Reply by skyzea1** [reply]
I'm on beta and it work thank you.

**Reply by chidimo** [reply]
I am having this exact problem for a week now. I'm using Google chrome.

Why not just give the option to use 2fa code?

This has been very frustrating.

**Reply by iskae** [reply]
Hi @chidimo 👋 

So sorry to hear that. Can you elaborate a bit more on what you have been experiencing with exact steps? 

Also, in the worst-case scenario, can you please try a fresh install?
